mathesar/api/serializers/shared_serializers.py

```diff
@@ -180,6 +180,14 @@ def validate(self, datetime_obj, display_format, serializer_field):
         return super().validate(datetime_obj, display_format, serializer_field)
 
 
+class DurationFormatValidator(TimeWithoutTimeZoneFormatValidator):
+    """
+    The validation is same as Time Without TimeZone as of now,
```

Comment:
This does not make sense. Durations can span weeks, months, days, and years. Times cannot.

Reply:
Fixed it, it was meant to be timestamp. I have changed the validation method anyway since we need a different error message, though the logic is similar.

Reply:
Also, I haven't restricted using a `weekdate` format string(2019-W17), though the duration value returned from the database follows RFC 3339 and does not return a weekdate, as the user might have a different requirement